packages/xo-server/src/utils.js

```diff
@@ -15,6 +15,8 @@ import { dirname, resolve } from 'path'
 import { utcFormat, utcParse } from 'd3-time-format'
 import { fromCallback, pAll, pReflect, promisify } from 'promise-toolbox'
 
+import { SimpleIdPattern } from './utils'
```

Comment:
Import from itself?

Reply:
If it's importing a type, use [`import type`](https://flow.org/en/docs/types/modules/).

Reply:
Anyway, we'll probably remove Flow one day so no need to spend time using it.

Personally, I'm mainly using it for documenting.